stock_internal_transfer_transit/__openerp__.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#   Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    'name': 'Stock internal transfer transit',
+    'version': '1.0',
+    'category': '',
+    'summary': "Process internal transfers using a dedicated "
+               "transfer stock location.",
```

Comment:
It seems to long to get a right display in the interface :

![image](https://cloud.githubusercontent.com/assets/7602419/7295402/c42926b6-e9b5-11e4-9653-26756c23349a.png)